crates/rethnet_eth/src/remote/client.rs

```diff
@@ -553,15 +557,17 @@ mod tests {
         client: RpcClient,
 
         // Need to keep the tempdir around to prevent it from being deleted
-        _cache_dir: TempDir,
+        // Only accessed when feature = "test-remote", hence the allow.
+        #[allow(dead_code)]
```

Comment:
```suggestion
        #[cfg(feature = "test-remote")]
```

How about?

Reply:
We need the the cache dir when `test-remote` is not set as well, but the attribute itself is only accessed in code gated behind `test-remote`.